straxen/plugins/event_area_per_channel.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import strax
+import straxen
+export, __all__ = strax.exporter()
+
+@export
+@strax.takes_config(
+    strax.Option('n_tpc_pmts', type=int,
+                 help='Number of TPC PMTs'),
+)
+class event_area_per_channel(strax.LoopPlugin):
+    """
+    Simple plugin that provides area per channel for main 
+    and alternative S1/S2 in the event. 
+    """
```

Comment:
Would it make sense to make this a savewhen = strax.SaveWhen.NEVER plugin as it's computation is trivial?

Reply:
It depends whether we will use it only here or in other plugins as well. My plan is to store it, as I want to use it for event level position re-reconstruction, so having it stored definitely helps. And there are plenty of studies where these areas can be extremely useful.

Reply:
Perhaps it's good to know what numbers we are talking about, what is the typical size / computation time of this datatype?

The fact that it's used elsewhere too is not a problem, in fact, `Peaks` as also `strax.SaveWhen.NEVER` and that is used on many different places.

Reply:
For background data that I tested, it is an order of 40-50 MB per run of 30 minutes, so it should not be too much space. As for peaks - the amount of data that has to be loaded and then not used is very different when one compares peaks vs peaklets and event areas vs peak(lets).